addons/pos_sale/models/sale_order.py

```diff
@@ -9,6 +9,7 @@ class SaleOrder(models.Model):
 
     pos_order_line_ids = fields.One2many('pos.order.line', 'sale_order_origin_id', string="Order lines Transfered to Point of Sale", readonly=True, groups="point_of_sale.group_pos_user")
     pos_order_count = fields.Integer(string='Pos Order Count', compute='_count_pos_order', readonly=True, groups="point_of_sale.group_pos_user")
+    amount_unpaid = fields.Monetary(string='Unpaid Amount', compute='_compute_amount_unpaid', store=False, help="The amount due from the sale order.")
```

Comment:
`store=False` is not useful here, it is the default behaviour of computed fields ;)

Reply:
Thx :)